tools/isobuild/package-namespace.js

```
@@ -155,7 +156,7 @@ export class PackageNamespace {
         // Do nothing. We might want to add some keys later, and we should err
         // on the side of backwards compatibility.
       }
-      if (size(compact([source.debugOnly, source.prodOnly, source.testOnly, source.devOnly])) > 1) {
+      if ([source.debugOnly, source.prodOnly, source.testOnly, source.devOnly].filter(Boolean).length > 1) {
```

ESLint

[error] Replace `[source.debugOnly,·source.prodOnly,·source.testOnly,·source.devOnly].filter(Boolean).length·>·1` with `⏎········[⏎··········source.debugOnly,⏎··········source.prodOnly,⏎··········source.testOnly,⏎··········source.devOnly,⏎········].filter(Boolean).length·>·1⏎······`

(prettier/prettier)